websurvey/lib/typewheels/responseStore.js

```diff
@@ -31,6 +31,16 @@ class ResponseStore {
       };
     }
   }
+
+  interpolationCheck(field, qa) {
+    const title = field.title;
+    const dynamicValue = f.getDynamicValue(qa, title);
```

Comment:
Why do we need this? Does interpolateField break in some way if there's no dynamic value?

Reply:
It does not – but I've added a test anyway to be doubly sure. 

The idea behind the check was just to avoid unnecessary interpolation so to say if there's nothing to interpolate don't run `interpolateField` and just give me the regular title. On second thoughts I don't know how much value it really adds and whether `interpolationCheck` is even needed when `interpolateField` does everything we need it to do. 

I could just remove the `dynamicValue` check from the method and have it run `interpolateField` in all cases.

Another approach could be to remove the `interpolationCheck` altogether and invoke `interpolateField` at the component level. Would there be any issue with moving the operation to the UI layer?

Reply:
Updated! Kept the method on `ResponseStore` for some abstraction but refactored it to a simple assignment. I removed the unit test too as discussed there's no need to test the biz logic twice